transaction-service/src/main/java/pl/adamboguszewski/transactionservice/application/Product.java

```diff
@@ -0,0 +1,35 @@
+package pl.adamboguszewski.transactionservice.application;
+
+public class Product {
+
+    private final String id;
+
+    private final long priceMultiplier;
+
+    private final long quantity;
+
+    private final long totalPrice;
+
+    public Product(String id, long priceMultiplier, long quantity, long totalPrice) {
```

Comment:
Use Long instead of long (generally in this project use Long instead of every int-related data type)